code/modules/clothing/under/accessories.dm

```diff
@@ -447,3 +447,31 @@
 /obj/item/clothing/accessory/allergy_dogtag/proc/on_examine(datum/source, mob/user, list/examine_list)
 	SIGNAL_HANDLER
 	examine_list += "The dogtag has a listing of allergies : [display]"
+
+/obj/item/clothing/accessory/pride
+	name = "rainbow pride pin"
+	desc = "A pin to show off your sexuality! Happy pride!"
```

Comment:
```suggestion
	desc = "A Nanotrasen Diversity & Inclusion Center-sponsored pin to show off your sexuality, reminding the crew of their unwavering commitment to equity, diversity, and inclusion."
```

It might be more interesting and thematic to have descs somehow relating to feigned corporate empathy